.github/workflows/gh-ci-tests.yaml

```diff
@@ -0,0 +1,58 @@
+name: mda_gh_ci_tests
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+    branches:
+      - develop
+  schedule:
+    # 3 am Tuesdays and Fridays
+    - cron: "0 3 * * 2,5"
+
+concurrency:
+  # Probably overly cautious group naming.
+  # Commits to develop/master will cancel each other, but PRs will only cancel
+  # commits within the same PR
+  group: "${{ github.ref }}-${{ github.head_ref }}-${{ github.workflow }}"
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    name: "examples-tests"
+    strategy:
+      fail-fast: False
+      matrix:
+        os: ['ubuntu']
+        python-version: [3.8]
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: True
+          use-mamba: True
+          python-version: ${{ matrix.python-version }}
+          miniforge-variant: Mambaforge
+          environment-file: environment.yml
+          activate-environment: mda-user-guide
+
+      - name: "environment information"
+        run: |
+          mamba info -a
+          mamba list
+      - name: "test notebooks"
+        run: |
+            git clone https://github.com/osmart/hole2 --depth 1
```

Comment:
I believe we don't pull from osmart's root branch because it's broken under some conditions. Please see/re-use from the MDA core CI as necessary: https://github.com/MDAnalysis/mdanalysis/blob/345d5d2d25a68df8ffac213fc3fe8bba2c9ec374/.github/actions/build-src/action.yaml#L39-L50

Reply:
I copied that section but removed all the `FC` references. I guess that workflow sets the Fortran Compiler on some other place, but the instance already comes with `gfortran` and it works just fine.